### DISCUSSION TITLE
Make tooltips clickable on mobile UI

### DISCUSSION THREAD

**Discussion by GrimmiMeloni** [solectrus/solectrus#2017] [Ideas]
The information in the tooltips is quite interesting and I like to take a look every now and then. On mobile however, the tooltips are basically not usable, as they rely on a non-existing hover event.

I'd like to suggest that the tooltips should be revealed by "clicking" event on mobile. I would consider it OK, to sacrifice the current click behavior on mobile, as these are merely shortcuts for the drop down selection (that is still available on mobile, too).

**Comment by ledermann** [comment]
Fully agree, created an issue:
https://github.com/solectrus/solectrus/issues/2018
